tensorflow/lite/CMakeLists.txt

```diff
@@ -623,14 +642,27 @@ set(_ALL_TFLITE_SRCS
   ${TFLITE_SOURCE_DIR}/schema/schema_utils.cc
   ${TFLITE_SOURCE_DIR}/schema/schema_generated.h
 )
-add_library(tensorflow-lite
-  ${_ALL_TFLITE_SRCS}
```

Comment:
If `TFLITE_BUILD_SHARED_LIB` is false, where is the `add_library` call?

Reply:
It is above, here https://github.com/tensorflow/tensorflow/pull/62933/files#diff-f22833f37d31fb773c988ab1a12b7c6a3265b1ee5eb3fe93c885a02a9f1d3d20R604, but I oversight the patch removes the `_ALL_TFLITE_SRCS` variable. Let us fix it.

Reply:
Hi @qukhan, we fixed the patch to correspond with the latest changes. Please re-review it. 
Thanks,

Reply:
@qukhan , we fixed the PR to corresponds to the latest changes. Please review it. 
Thanks,